automatic/screencloud/update.ps1

```diff
@@ -0,0 +1,31 @@
+Import-Module au
+
+$source = "https://screencloud.net/pages/download"
+$pattern = "ScreenCloud-(.+)-x86.msi"
+
+function global:au_BeforeUpdate {
+    Get-RemoteFiles -Purge -NoSuffix
+}
+
+function global:au_SearchReplace {
+    @{
+        ".\legal\VERIFICATION.txt" = @{
+            "(?i)(\s+x32:).*"      = "`${1} $($Latest.URL32)"
+            "(?i)(\checksum32:).*" = "`${1} $($Latest.Checksum32)"
+        }
+    }
+}
+
+function global:au_GetLatest {
+    $res = Invoke-WebRequest -Uri $source -UseBasicParsing
+
+    $url = $res.Links | ? href -match $pattern | % href
```

Comment:
Is there always a guarantee that only 1 url will match?
Would be better to change `% href` to `select -First 1 -Expand href`